desktop-app/build.js

```diff
@@ -0,0 +1,93 @@
+const createDMG = require("electron-installer-dmg");
+const { signAsync } = require("@electron/osx-sign");
+const electronInstaller = require("electron-winstaller");
```

Comment:
Here there are some esLint errors because of the order of imports. Can you fix them please. You can run `npm run lint` to see them.
Also can you please change the `.eslintignore` to not include the built apps. 
You can also add `/* eslint-disable no-console */` for the whole file `build.js`